### ISSUE TITLE
Indentation errors but why?

### ISSUE THREAD

**Issue by IschaGast** [ahmednuaman/grunt-scss-lint#105]
This is in my settings file
``` 
  Indentation:
    enabled: true
    allow_non_nested_indentation: false
    character: space # or 'tab'
    width: 4

  SelectorFormat:
    enabled: true
    convention: hyphenated_BEM # hyphenated_lowercase # or 'strict_BEM', or 'hyphenated_BEM', or 'snake_case', or 'camel_case', or a regex pattern
```

But this code below still gives an indentation error but don't see why?
```
static/scss/base/_grid.scss:17 [W] Indentation: Line should be indented 0 spaces, but was indented 4 spaces
```
``` CSS
.grid {
    display: table;
    table-layout: fixed;
    width: 100%;
}
    .grid__unit {
        display: table-cell;
        padding-left: 10px;
        padding-right: 10px;
    }    
```

**Comment by ahmednuaman** [comment]
Hi @IschaGast, this is probably isn't the place for this, it'll be a better idea to open it again @brigade's repo: https://github.com/brigade/scss-lint
